zerver/views/registration.py

```diff
@@ -164,24 +183,7 @@ def accounts_register(request: HttpRequest) -> HttpResponse:
                         require_ldap_password = isinstance(backend, ZulipLDAPAuthBackend)
                         break
                     except TypeError:
-                        # Let the user fill out a name and/or try another backend
-                        form = RegistrationForm(realm_creation=realm_creation)
-        elif prereg_user.full_name:
-            if prereg_user.full_name_validated:
-                request.session['authenticated_full_name'] = prereg_user.full_name
-                name_validated = True
-                form = RegistrationForm({'full_name': prereg_user.full_name},
-                                        realm_creation=realm_creation)
-            else:
-                form = RegistrationForm(initial={'full_name': prereg_user.full_name},
-                                        realm_creation=realm_creation)
-        elif 'full_name' in request.POST:
-            form = RegistrationForm(
-                initial={'full_name': request.POST.get('full_name')},
-                realm_creation=realm_creation
-            )
-        else:
-            form = RegistrationForm(realm_creation=realm_creation)
+                        break
```

Comment:
So I think this change pushes away from the possibility that we could eventually support multiple LDAP backends, since it'll give up after the first one fails to work.  Not sure we care about that use case right now, but it's something we'll likely eventually support.

Reply:
That's probably OK; we make that assumption in like 5 other places.

Reply:
When adding more ldap backends, these would only have to be changed to ``continue``, to try the next backend in the list, right?